proto/blockchain_service.proto

```diff
@@ -109,6 +109,17 @@ message GetProjectsInfoResponse {
     repeated ProjectInfo projects = 1;
 }
 
+message GetInvestmentDetailsRequest {
+    string investor_wallet = 1;
+    string project_tx_hash = 2;
+}
+
+message GetInvestmentDetailsResponse {
+    string investor_balance = 1;
+    string investment_amount = 2;
+    string total_funds_raised = 3;
+}
+
```

Comment:
@githinho @josipkoncurat check these proto changes for new grpc route: GetInvestmentDetails
This will be called from backend to provide investment data for @mdrusko1 's [task](https://dev.azure.com/ampnetio/AMPnet%20Crowdfunding/_workitems/edit/626)